src/Symfony/Component/ClassLoader/ClassMapGenerator.php

```diff
@@ -134,7 +134,7 @@ private static function findClasses($path)
                     }
 
                     if ($isClassConstant) {
-                        continue;
+                        break;
```

Comment:
Are you sure about this change? `continue` relates to the `for` loop while `break` only leaves the `switch`.